src/validations/rules/ssp.sch

```diff
@@ -5337,6 +5394,16 @@
             doc:assertion="import-profile-has-href-attribute"
             doc:context="oscal:import-profile"
             id="import-profile-has-href-attribute-diagnostic">The import-profile element lacks an href attribute.</sch:diagnostic>
+        <sch:diagnostic
+            doc:assertion="technical-control-exists"
+            doc:context="oscal:control-implementation"
+            id="technical-control-exists-diagnostic">The technical control implementation <sch:value-of
+                select="@control-id" /> does not exist in the SSP document.</sch:diagnostic>
```

Comment:
The context of the associated assertion is `oscal:control-implementation`. Why is `@control-id` being referenced?

Reply:
fixed